Src/Support/Google.Apis.Auth/GoogleJsonWebSignature.cs

```diff
@@ -124,7 +112,37 @@ private ValidationSettings(ValidationSettings other)
             /// </summary>
             public TimeSpan ExpirationTimeClockTolerance { get; set; } = TimeSpan.FromSeconds(0);
 
+            /// <summary>
+            /// CertificateCache for testing purposes.
+            /// If null, the true CertificateCache will be used.
```

Comment:
What do you mean by "the true CertifcateCache"? A default from somewhere? Can you say where from?